api/src/main/java/com/vimeo/networking2/config/RetrofitSetupModule.kt

```diff
@@ -17,47 +17,19 @@ import java.util.concurrent.TimeUnit
  */
 object RetrofitSetupModule {
 
-    private const val USER_AGENT_HEADER = "User-Agent"
     private const val AUTHORIZATION_HEADER = "Authorization"
+    private const val USER_AGENT_HEADER = "User-Agent"
+    private const val USER_AGENT_HEADER_VALUE = "Kotlin VimeoNetworking/$SDK_VERSION"
```

Comment:
The current user agent is `VimeoNetworking/version (Java)`. Would it be a good idea to keep the same pattern and change this to `VimeoNetworking/$SDK_VERSION (Kotlin)`?